metriken-exposition/src/snapshot.rs

```diff
@@ -82,3 +79,37 @@ impl Snapshot {
         rmp_serde::encode::to_vec(val)
     }
 }
+
+#[cfg(feature = "parquet")]
+impl From<Snapshot> for HashedSnapshot {
+    fn from(snapshot: Snapshot) -> Self {
+        let mut counters: HashMap<String, u64> = HashMap::new();
```

Comment:
You can build these using HashMap::from_iter() instead:
`let counters: HashMap<String, u64> = HashMap::from_iter(snapshot.counters);`